src/migrations/20240124082206-monitoring-data.js

```diff
@@ -0,0 +1,547 @@
+          allowNull: true,
+        },
+        statusId: {
+          type: Sequelize.INTEGER,
+          allowNull: true,
+        },
+        startDate: {
+          type: Sequelize.DATEONLY,
+          allowNull: true,
+        },
+        endDate: {
+          type: Sequelize.DATEONLY,
+          allowNull: true,
+        },
+        reviewType: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        reportDeliveryDate: {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        outcome: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        hash: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        sourceCreatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        sourceUpdatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        sourceDeletedAt: {
+          allowNull: true,
+          type: Sequelize.DATE,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        deletedAt: {
+          allowNull: true,
+          type: Sequelize.DATE,
+        },
+      }, {
+        transaction,
+      });
+
+      await queryInterface.sequelize.query(`
+          CREATE UNIQUE INDEX "MonitoringReviews_reviewId_deletedAt"
+          ON "MonitoringReviews"
+          ("reviewId", "deletedAt");
+      `, { transaction });
+
+      await queryInterface.sequelize.query(`
+          ALTER TABLE "MonitoringReviews"
+          ADD CONSTRAINT "MonitoringReviews_reviewId_deletedAt_unique"
+          UNIQUE USING INDEX "MonitoringReviews_reviewId_deletedAt";
+      `, { transaction });
+
+      await queryInterface.sequelize.query(`
+          CREATE INDEX "MonitoringReviews_statusId"
+          ON "MonitoringReviews"
+          ("statusId")
+          WHERE "deletedAt" IS NULL;
+      `, { transaction });
+
+      await queryInterface.sequelize.query(`
+          CREATE INDEX "MonitoringReviews_reviewId_statusId"
+          ON "MonitoringReviews"
+          ("reviewId", "statusId")
+          WHERE "deletedAt" IS NULL;
+      `, { transaction });
+
+      await queryInterface.createTable('MonitoringReviewGrantees', {
+        id: {
+          type: Sequelize.INTEGER,
+          allowNull: false,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        reviewId: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        granteeId: {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        },
+        createTime: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updateTime: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updateBy: {
+          allowNull: false,
+          type: Sequelize.DATE,
```

Comment:
updateBy is supposed to be a DATE? Seeders make it look like maybe it should be a string, e.g. "Support Team"
(cc @hardwarehuman)